src/MATLAB.jl

```diff
@@ -41,23 +41,113 @@ end
 include("mxbase.jl")
 include("mxarray.jl")
 include("matfile.jl")
-
 include("mstatements.jl")
 include("engine.jl")
 include("matstr.jl")
 
 function __init__()
+
+    # load libraries
+
+    libmx[] = Libdl.dlopen(joinpath(matlab_lib_path, "libmx"), Libdl.RTLD_GLOBAL)
+    libmat[] = Libdl.dlopen(joinpath(matlab_lib_path, "libmat"), Libdl.RTLD_GLOBAL)
+    libeng[] = Libdl.dlopen(joinpath(matlab_lib_path, "libeng"), Libdl.RTLD_GLOBAL)
```

Comment:
maybe add `@assert libmx[] != C_NULL` 
or
 `libmx[] == C_NULL && error("The MATLAB library path could not be found.")`

?